aws/cloudformation/bootstrap_frontend.sh.erb

```diff
@@ -19,3 +19,16 @@ ps aux | grep "cluster worker" | grep dashboard | head -n 3 | tr --squeeze-repea
 # Increase EC2 instance metadata service retries/timeouts.
 export AWS_METADATA_SERVICE_TIMEOUT=30
 export AWS_METADATA_SERVICE_NUM_ATTEMPTS=30
+
+# Smoke test: Make sure server is responding to HTTP requests.
+CURL_OPTS=(
+  -H 'x-forwarded-proto: https'
+  --silent --show-error --fail
+  --retry 10 --retry-connrefused --retry-max-time 60
+)
+curl --fail-early \
+  "${CURL_OPTS[@]}" -H 'Host: <%=subdomain%>' localhost:80 \
+  --next \
+  "${CURL_OPTS[@]}" -H 'Host: <%=studio_subdomain%>' localhost:80/courses \
```

Comment:
Should we align this with the path that our load balancer uses to verify an instance in a Target Group is healthy (`/health_check`)?

Reply:
Very good idea! Yes, I forgot about the health-check layers that are already in place, it will simplify this tremendously to use that endpoint directly.

Reply:
Updated in 81db544361723601bb29ea9b99b899543b57a816